src/accountingservice/kafka/consumer.go

```diff
@@ -62,12 +63,7 @@ func (g *groupHandler) ConsumeClaim(session sarama.ConsumerGroupSession, claim s
 			if err != nil {
 				return err
 			}
-
-			g.log.WithFields(logrus.Fields{
-				"orderId":          orderResult.OrderId,
-				"messageTimestamp": message.Timestamp,
-				"messageTopic":     message.Topic,
-			}).Info("Message claimed")
+			g.log.LogAttrs(session.Context(), slog.LevelInfo, "Message claimed", slog.String("orderId", orderResult.OrderId), slog.String("messageTimestamp", message.Timestamp.String()), slog.String("messageTopic", message.Topic))
```

Comment:
The notice that `session.Context()` does NOT contain the trace context information as `NewOTelInterceptor` does not pass the context created in `oi.tracer.Start`. Therefore the logs would not contain the `TraceID` and `SpanID` fields.

Reply:
Okay, so should I just use a new context like `context.TODO()`?

Reply:
Okay